polynote-frontend/polynote/data/data.ts

```diff
@@ -84,6 +84,19 @@ export class IvyRepository extends RepositoryConfig {
         return 0;
     }
 
+    // enable parsing when copy-pasting configurations
+    toJSON(): any {
+        return {
+            type: "ivy",
+            resolver: {
```

Comment:
Could this be simplified using `resolver: ...this`? 

If that works, it's really tempting to lift this all up into `RepositoryConfig` 😅